ThingIFSDK/ThingIFSDK/OnboardWithVendorThingIDOptions.swift

```diff
@@ -41,3 +41,32 @@ public struct OnboardWithVendorThingIDOptions {
         self.layoutPosition = position
     }
 }
+
+extension OnboardWithVendorThingIDOptions: JsonObjectCompatible {
+
+    internal init(_ jsonObject: [String : Any]) throws {
+        // This method may not use so this method is not tested.
```

Comment:
> I didn't expected to see comment like this.

Maybe it's a symptom of wrong decision. 
If we add test this time, someone will skip test in future.